projects/aws/upgrader/Makefile

```diff
@@ -0,0 +1,33 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+
+REPO=upgrader
+REPO_OWNER=aws
+
+GIT_TAG=v$(RELEASE_BRANCH)-$(shell yq e ".releases[] | select(.branch==\"${RELEASE_BRANCH}\").number" $(BASE_DIRECTORY)/EKSD_LATEST_RELEASES)
+
+HAS_RELEASE_BRANCHES=true
+
+BUILDSPEC_VARS_KEYS=RELEASE_BRANCH
+BUILDSPEC_VARS_VALUES=SUPPORTED_K8S_VERSIONS
+BUILDSPEC_DEPENDS_ON_OVERRIDE=containerd_containerd_linux_amd64 containerd_containerd_linux_arm64 kubernetes_sigs_cri_tools
```

Comment:
i dont love you had to do this, but i understand why and im impressed you got it, thats really some gnarly code...